src/_includes/layouts/project.njk

```diff
@@ -44,6 +44,11 @@
             </ul>
         </div>
     </inclusive-disclosure>
+    {% if logo %}
+    <div class="logo">
+    <img src="{{ logo }}" alt="" />
```

Comment:
instead of leaving alt as a blank, I think recommended practice is to set aria-hidden="true" on icons.

Reply:
As this is an `img` tag, an empty `alt` is probably the right approach here (as opposed to an `<svg>` element). I also tend to avoid using `aria-hidden` on elements that aren't in an interactive parent; so, for example, an icon within a link or button should use `aria-hidden="true"` but as this is just an image on the page an empty `alt` tag preferable.

See here: https://www.boia.org/blog/images-that-dont-need-alternative-text-still-need-alt-attributes

Reply:
Thanks for the article! That makes sense.